autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.css

```diff
@@ -13,25 +14,35 @@
 .column-wrapper {
     position: relative;
     width: 100%;
+    height: 100%;
     flex-grow: 1;
     margin-bottom: 8px;
 }
 
 .course-select-column {
     position: absolute;
+    display: flex;
+    flex-direction: column;
     top: 0px;
     bottom: 0px;
     overflow-y: auto;
     width: 100%;
+    height: 100%;
 }
 
 .row {
-    padding-bottom: 8px;
+    padding-top: 8px;
     padding-right: 8px;
+    box-sizing: border-box;
+}
+
+.expanded-row {
+    overflow-y: hidden;
+    min-height: 500px;
 }
 
-.row:nth-last-of-type(2) {
-    padding-bottom: 0px;
+.expanded-row-overflow {

```

Comment:
I would rename this to `expanded-row-small` since it's applied to small/short cards that fit within the 500px limit